strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/xml/configuration/repository/remote/MutableNpmRemoteRepositoryConfiguration.java

```diff
@@ -0,0 +1,40 @@
+package org.carlspring.strongbox.xml.configuration.repository.remote;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.carlspring.strongbox.xml.repository.remote.CustomRemoteRepositoryConfiguration;
+import org.carlspring.strongbox.xml.repository.remote.MutableRemoteRepositoryConfiguration;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "npm-remote-repository-configuration")
+public class MutableNpmRemoteRepositoryConfiguration extends MutableRemoteRepositoryConfiguration
+{
+    @XmlAttribute(name = "last-cnahge-id")
```

Comment:
There's a typo in both the XML attribute name and the Java field.

Reply:
yeah, I just copypaste it everywhere 
will fix it, thanks